tp-app-payment-gateway-dev.yaml

```diff
@@ -23,6 +23,15 @@ paymentgateway:
     keysecret: ${PAYPAL_KEY_SECRET:EMuFpv1mkjSavIVveEdq3zxLMjuDl8SOPYLjl8dgtRhDqI2zXK-YT3l95jpPbk-uS1mFmyM6r1RnTvc5}
```

Comment:
donot commit secrets, please remove

Reply:
this was already there and i think is being used